Puppetfile

```diff
@@ -1,12 +1,9 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
-mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
-mod 'katello/pulp',                  :git => 'https://github.com/theforeman/puppet-pulp'
-mod 'katello/qpid',                  :git => 'https://github.com/theforeman/puppet-qpid'
-
-# A fork that marks metadata as compatible with puppet 5
-# https://github.com/thias/puppet-squid3/pull/59
-mod 'thias-squid3',                  :git => 'https://github.com/theforeman/puppet-squid3', :ref => 'allow-puppet-5'
```

Comment:
This one is still needed because we use a fork. I've started to work on replacing this module but that's still WIP.

Reply:
Heh, I just used the `pin_modules` task. It may need to be smarter.

Reply:
Perhaps if there's a ref then it should keep it.